subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/DependencyUnresolvedModuleIntegrationTest.groovy

```diff
@@ -219,10 +219,10 @@ class DependencyUnresolvedModuleIntegrationTest extends AbstractHttpDependencyRe
     }
 
     @Unroll
-    def "fails build if HTTP connection #reason when resolving metadata"() {
+    def "fails build and #abortDescriptor repository search if HTTP connection #reason when resolving metadata"() {
         given:
         MavenHttpRepository backupMavenHttpRepo = new MavenHttpRepository(server, '/repo-2', new MavenFileRepository(file('maven-repo-2')))
```

Comment:
Looks like `missing executer.withArgument("-D${SOCKET_TIMEOUT_SYSTEM_PROPERTY}=1000")` here.

Reply:
That is configured in `setup()`. Do I need something additional for this test?
Note that I didn't change much in this test.

Reply:
Nothing else I think.